### PR TITLE
Add model validations and spec

### PR DESCRIPTION
**Pull request by Aqurds**
The following features were added,

**Add the following validations:**

> **For the User model:**
> Name must not be blank.
> PostsCounter must be an integer greater than or equal to zero.

> **For the Post model:**
> Title must not be blank.
> Title must not exceed 250 characters.
> CommentsCounter must be an integer greater than or equal to zero.
> LikesCounter must be an integer greater than or equal to zero.

**Add unit specs for all of your models' methods and validations.**

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it { should validate_presence_of(:name) }
+  it { should validate_numericality_of(:postCounter).only_integer.is_greater_than_or_equal_to(0) }
+end
```
Comment (rloterh) [inline spec/models/user_spec.rb:6]: - [x] Good job on validations and adding test spec for `user` and `post` models. However, let's also endeavor to add test specs for `comment` and `like` models.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Omar, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project, but you are almost there! 
🏁 🌴 🚦 🚗 🏫 🏡 _🏃

## Highlights 👍  
- Validations properly implemented!
- ReadMe looks good!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
Let's make the ReadMe stand out even better by including instructions under the _Getting Started_ section, on how to run test cases for the app, for instance, using the command `rspec`.

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/user_spec.rb (hunk 1 above)
Comment (Aqurds) [reply spec/models/user_spec.rb:6]: Unit specs added for comment and like models. Thanks :)

**Review by aamirkhan2478 — APPROVED** [review]
Hi @Aqurds, 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly :+1: 
✅ No linter error :+1: 
✅ Good commit messages :+1: 
✅ Good descriptive PR :+1: 
✅ Well done :clap: you did all changes given by the previous reviewer :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**